packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelForm.tsx

```diff
@@ -139,6 +171,14 @@ export default function ColumnModelForm(props: ColumnModelFormProps) {
     return {}
   }, [validationErrors])
 
+  const showErrorOnMaxSizeField = !!errorsByField['maximumSize']
+
+  const helperTextForMaxSizeField = errorsByField['maximumSize']
+    ? errorsByField['maximumSize']
+    : isSmallerThanRecommendedSize && recommendedSize
+    ? `Recommended size is at least ${recommendedSize}`
+    : ''
```

Comment:
Is space created for this component when this is the empty string? If not, then what you have is fine
```suggestion
    : undefined
```